src/styles.js

```diff
@@ -77,8 +78,10 @@ export function installStyles(doc, cssText, cb, opt_isRuntimeCss) {
  * If the body is not yet available (because our script was loaded
  * synchronously), polls until it is.
  * @param {!Document} doc The document who's body we should make visible.
+ * @param {Array<Promise>=} extensions Special extensions which must load
+ *     before the body can be made visible
  */
-export function makeBodyVisible(doc) {
+export function makeBodyVisible(doc, extensions) {
   let interval;
```

Comment:
I feel like this function now really needs a global timeout. 8 seconds is what we typically use.

Too worried that some error would lead to all(extensions) never resolving.